samza-api/src/main/java/org/apache/samza/task/TaskContext.java

```diff
@@ -58,10 +58,9 @@
   /**
    * Method to allow user to return customized context
```

Comment:
Documentation: Shouldn't refer to user in third person in public APIs. Also, "Method to" is redundant.